### ISSUE TITLE
Autotype position off when pausing

### ISSUE THREAD

**Issue by glitchroy** [JujuAdams/scribble#15]
Hey, it's me again, with a problem similar to #11, in fact, it may be the very same problem, but in v5.0.X.  

I tried forking scribble and adding a clean way to "interrupt" the autotyper by adding a new property, but there is a consistent issue that the "pausing" happens one character too late.  
I'm not sure, but it could very well be that the event triggers one character later, since the script call happens in the middle of the autotype processing and is directly referencing it. I don't think it's a shader issue this time. I tried looking for rounding errors, too, but I don't think that's it either.

Regardless, I'm not confident enough to just put a -=1 somewhere in there, so I was hoping you could check out the patch here:
https://github.com/glitchroy/scribble/commit/815d129b27fc0584201d9c3d064c27a64c73bc65#diff-ea72a6c3e7f97ed30382838bacf15089
It's not about accepting the feature in the first place, since I did add this behaviour without forking scribble previously (and because it didn't work, I started taking a "cleaner" approach). This is just a convient way to see the problem, I think.  

Thanks!

**Comment by JujuAdams** [comment]
yo yo yo, thanks for your bug report as always.

I did think about putting in an AUTOTYPE_PAUSE property to the text elements (though can I just say "INTERRUPT" sounds so much cooler) but I decided against it. My concern is, as usual, not wanting to overburden the library. I'll reflect on that decision.

In the meantime, I'll check the off-by-one-ness of the event positioning. This could be a bug elsewhere that's having a knock-on effect.
